src/node/blockstorage.cpp

```diff
@@ -834,6 +835,9 @@ void ThreadImport(ChainstateManager& chainman, std::vector<fs::path> vImportFile
         // -reindex
         if (fReindex) {
             int nFile = 0;
+            // Map of disk positions for blocks with unknown parent (only used for reindex);
```

Comment:
nit (no need to re-push for that): "(only used for reindex)" is obvious and could be dropped now that this got moved into a code block conditional on `fReindex`.